### PR TITLE
chicken: update to 4.11.0

### PR DESCRIPTION
**Pull request by rkallos**

### CODE REVIEW

File: srcpkgs/chicken/template [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Template file for 'chicken'
 pkgname=chicken
-version=4.9.0.1
-revision=4
+version=4.11.0
+revision=5
```
Comment (Vaelatern) [inline srcpkgs/chicken/template]: No, on update of version, revision gets returned to 1

**Comment by Vaelatern** [comment]
Please see the build error, it tells you what to do.

**Comment by rkallos** [comment]
Whoops! Should be all fixed now.

**Comment by Vaelatern** [comment]
Much better!

**Pull request merged**